src-ghc/Pact/Interpreter.hs

```diff
@@ -305,7 +305,7 @@ interpret runner evalEnv terms = do
   return $! EvalResult
     terms
     (map (elideModRefInfo . toPactValueLenient) rs)
-    logs pactExec gas modules txid gasLogs (_evalEvents state) warnings
+    logs pactExec (milliGasToGas microGas) modules txid gasLogs (_evalEvents state) warnings
```

Comment:
Shall `EvalResult` still carry the `Gas` or rather the `MilliGas`?

Reply:
`Gas` otherwise we break Chainweb downstream.